torch/csrc/jit/pass_manager.cpp

```diff
@@ -0,0 +1,16 @@
+#include <torch/csrc/jit/pass_manager.h>
+
+namespace torch {
+namespace jit {
+
+std::vector<Pass>& getPasses() {
+  static std::vector<Pass> passes;
+  return passes;
+}
+
+RegisterPass::RegisterPass(Pass p) {
+  getPasses().emplace_back(std::move(p));
+}
+
```

Comment:
shall we also expose an interface to remove a custom pass? it would be nice so that backend can control registering some passes and disable some custom passes they registered if necessary.

Reply:
In traditional compilers is usually handled by having target hooks, which any pass can check. If some property is disabled on the current target, a pass just becomes no-op. That is, it remains in the pipeline, but its behavior depends on these target-specific properties. We can do something similar here I guess.